test/specs/upgrade/upgrade.ts

```diff
@@ -0,0 +1,89 @@
+import { spawnSync } from 'child_process';
+import WikibaseApi from 'wdio-wikibase/wikibase.api.js';
+import { versions } from '../../suites/upgrade/versions.js';
+import { getTestString } from 'wdio-mediawiki/Util.js';
+import assert from 'assert';
+
+describe( 'Wikibase upgrade', function () {
+	let oldItemID: string;
+
+	before( async () => {
```

Comment:
This before hook works in concert with what happens in `test/suites/upgrade/upgrade.conf.ts`. See comments.